Resources/Prototypes/Entities/Objects/Devices/door_remote.yml

```
@@ -0,0 +1,116 @@
+- type: entity
+  parent: BaseItem
+  id: DoorRemoteDefault
+  name: door remote
+  description: A gadget which can open and bolt doors remotely
+  abstract: true
+  components:
+  - type: Sprite
+    sprite: Objects/Devices/door_remote.rsi
+    netsync: false
+  - type: Access
+  - type: DoorRemote
+
+- type: entity
+  parent: DoorRemoteDefault
+  id: DoorRemoteCommand
+  name: command door remote
+  components:
+  - type: Sprite
+    layers:
+    - state: door_remotebase
+    - state: door_remotecommand
+  - type: Access
+    tags: 
+    - Captain
+    - HeadOfPersonnel
+    - Command
+
+- type: entity
+  parent: DoorRemoteDefault
+  id: DoorRemoteSecurity
+  name: security door remote
+  components:
+  - type: Sprite
+    layers:
+    - state: door_remotebase
+    - state: door_remotesecurity
+  - type: Access
+    tags:
+    - HeadOfSecurity
+    - Security
+
+- type: entity
+  parent: DoorRemoteDefault
+  id: DoorRemoteService
+  name: service door remote
+  components:
+  - type: Sprite
+    layers:
+    - state: door_remotebase
+    - state: door_remoteservice
```

In case sloth's comment on colors wasn't clear: instead of using the different coloured states for each type of remote, it would be better to have a single white state and then add a `color` attribute in yaml that just re-colors that white layer. That way its very easy to add new colors without having to edit any sprites.

e.g., if you had a single white overlay called something like `door_remoteoverlay` you could just replace this bit of yaml with something like
```suggestion
  - type: Sprite
    layers:
    - state: door_remotebase
    - state: door_remoteoverlay
      color: e8ab28 # or whatever actually matches the color, I just picked this at random.
```